### PR TITLE
Add add-ons page

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

**Comment by balloob** [comment]
Can we change the first two menu items to `Installs`, `Stats` so it better fits on 1 line on mobile?

**Comment by ludeeus** [comment]
Adjusted headers and migration

File: site/src/analytics-addons.ts [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+      "https://github.com/hassio-addons/repository/blob/master/zerotier/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/zerotier/icon.png",
+  },
+  a0d7b954_zwavejs2mqtt: {
+    name: "Z-Wave JS to MQTT",
+    documentation:
+      "https://github.com/hassio-addons/repository/blob/master/zwavejs2mqtt/DOCS.md",
+    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/zwavejs2mqtt/icon.png",
+  },
+};
```
Comment (balloob) [inline site/src/analytics-addons.ts:323]: Can we add the ESPHome Repo.

File: site/src/analytics-addons.ts [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+  @state() private _filter: string = "";
+
+  @state() private _addons?: AddonData[];
+
+  @state() private _currentTableSize = 30;
+  @state() private _currentTablePage = 0;
+
+  protected firstUpdated(_changedProperties: PropertyValues) {
+    super.firstUpdated(_changedProperties);
+    this._filter = "";
```
Comment (balloob) [inline site/src/analytics-addons.ts:340]: Not needed as it's like that in the constructor.

File: site/src/analytics-addons.ts (hunk 1 above)
Comment (ludeeus) [reply site/src/analytics-addons.ts:323]: We can do that when it's shipping with the supervisor.

File: site/src/analytics-addons.ts [hunk 3]
```diff
@@ -0,0 +1,672 @@
...
+              <td>
+                <a
+                  title="Documentation"
+                  href="${ADDONS[entry.slug].documentation ||
+                  `https://github.com/home-assistant/addons/blob/master/${entry.slug.replace(
+                    "core_",
+                    ""
+                  )}/DOCS.md`}"
+                  target="_blank"
+                  rel="noreferrer"
```
Comment (balloob) [inline site/src/analytics-addons.ts:427]: We're not leaking personal info so you don't need to disable the referrer.

File: site/src/analytics-addons.ts [hunk 4]
```diff
@@ -0,0 +1,672 @@
...
+              `
+            : ""
+        }
+          </div>
+      </div>
+      <table>
+        <tr class="table-header">
+          ${!this.isMobile ? html`<th class="idx"></th>` : ""}
+          <th>Add-on</th>
+          <th class="installations">Installations</th>
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply site/src/analytics-addons.ts:410]: it will grow in < 24h when we have % there as well

File: site/src/analytics-integrations.ts [hunk 5]
```diff
@@ -200,15 +200,20 @@ export class AnalyticsIntegrations extends LitElement {
...
-        (${+(
-          (100 * this.currentData.reports_integrations || 0) /
-          this.currentData.extended_data_from
-        ).toFixed(2)}%)
-        installations have chosen to share their used integrations
-      </div>
+      ${
+        this.currentData.reports_integrations
+          ? html`<div class="footer">
+              ${this.currentData.reports_integrations || "Unkown"} of
```
Comment (balloob) [inline site/src/analytics-integrations.ts:206]:
```suggestion
              ${this.currentData.reports_integrations || "Unknown"} of
```

File: site/src/analytics-addons.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (balloob) [reply site/src/analytics-addons.ts:410]: Right, but on mobile it's going to have to reserve space for the whole word.

image

**Pull request merged**